### PR TITLE
Color-hex.com presence

### PR DESCRIPTION
**Pull request by MiepHD**
I have revised my presence and open a request again for color-hex.com.

### CODE REVIEW

File: websites/C/color-hex/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "Laila",
+    "id": "636229819620655134"
+  },
+  "service": "color-hex",
+  "description": {
+    "en": "Color-hex gives information about colors.",
+    "de": "Color-hex gibt Informationen über Farben."
+  },
+  "url": "www.color-hex.com",
+  "version": "0.1",
```
Comment (br1ss) [inline websites/C/color-hex/dist/metadata.json:12]: only 1.0.0 version

Comment (MiepHD) [reply websites/C/color-hex/dist/metadata.json]: I changed it.

**Review by rxri — CHANGES_REQUESTED** [review]
image
it's still a thing.
and version `1.0.0` not `1.0`

**Comment by MiepHD** [comment]
Am I only allowed to do one DOM reading?

**Comment by ririxidev** [comment]
> Am I only allowed to do one DOM reading?

You NEED to do more DOM readings or provide more functionality.

**Comment by MiepHD** [comment]
I have several dom readings and cover nearly all of the funtionalities of the website. I had asked only because i was worried that i am not allowed to have more than one reading.

**Comment by ririxidev** [comment]
You only do that 2 times

**Comment by MiepHD** [comment]
Yes for two pages it made sense to get infos from within the pages but for most pages it is enough to read the url or the title

**Comment by ririxidev** [comment]
So this presence will not be accepted until you will not get more DOM readings or so 😃

**Comment by MiepHD** [comment]
Should I use DOM readings instead of using title and path to make the code more complicated?

**Comment by MiepHD** [comment]
I can't find a rule in the Guidelines which says that I must have a certain number of DOM Readings. Could you please tell me where I can see this rule? Why do I have to have a certain amount of DOM Readings?

**Comment by ririxidev** [comment]
This presence is just qualified for low quality at this point since presence is more likely "static". And I'm not going to continue this conversation anymore since I already told you what to do.

**Comment by MiepHD** [comment]
Did you visit the site at any time?? There are no more functions on the site! Because there are no more functions I can't add functions to the presence.

**Comment by MiepHD** [comment]
> This presence is just qualified for low quality at this point since presence is more likely "static". And I'm not going to continue this conversation anymore since I already told you what to do.

It's a mostly static site. So isn't it logically that the presence is also like that?

**Comment by MiepHD** [comment]
@br1ss What do you think about this presence? Has it enough functionality?

**Comment by ririxidev** [comment]
Sorry but `br1ss` is not Presence Reviewer. If you can't provide more functionality it means that site is not qualified. I'm sorry but we are accepting only "good" presences, not presences like yours rn.
If you want to report my actions, please do to Staff Coordinator (Bas950), his discord is Bas950#0950.

**Pull request closed without merging**